cloudpickle/cloudpickle.py

```diff
@@ -966,14 +985,32 @@ def _save_parametrized_type_hint(self, obj):
                 initargs = (Final, obj.__type__)
             elif type(obj) is type(ClassVar):
                 initargs = (ClassVar, obj.__type__)
-            elif type(obj) in [type(Union), type(Tuple), type(Generic)]:
-                initargs = (obj.__origin__, obj.__args__)
+            elif type(obj) is type(Generic):
+                if sys.version_info < (3, 5, 2):  # pragma: no cover
```

Comment:
Maybe it's possible to ask for a specific minor version on github actions?

Reply:
From a quick skim I have not seen anything about minor versions... :(

Reply:
but have you tried to update the yaml file directly to point to 3.5.2 instead of 3.5? It's worth a blind try.

Reply:
let's try.

Reply:
Seems like it's not working. Maybe https://github.com/goanpeca/setup-miniconda?